core/app/models/spree/app_configuration.rb

```diff
@@ -47,6 +47,7 @@ class AppConfiguration < Preferences::Configuration
     preference :disable_sku_validation, :boolean, default: false # when turned off disables the built-in SKU uniqueness validation
     preference :expedited_exchanges, :boolean, default: false # NOTE this requires payment profiles to be supported on your gateway of choice as well as a delayed job handler to be configured with activejob. kicks off an exchange shipment upon return authorization save. charge customer if they do not return items within timely manner.
     preference :expedited_exchanges_days_window, :integer, default: 14 # the amount of days the customer has to return their item after the expedited exchange is shipped in order to avoid being charged
+    preference :favicon, :string, default: 'favicon.ico'
```

Comment:
@mad-eel let's drop the preference at all, it should be only manageable via Admin UI sorry I didn't tell you that before, we're moving away from app preferences to Store manageable configuration in the UI